_components/icon-list/icon-list.md

```diff
@@ -22,9 +22,9 @@ title: Icon list
 type: component
 variants:
   - variant: "`.usa-icon-list--[color]`"
-    description: Change the color of all the list's icons by updating [color] to any one of the theme colors listed on the [color utilities](/utilities/color) page.
-  - variant: "`usa-icon-list--size-[size]`"
-    description: "Change the size of the text and icon by updating [size] to a theme font size: `2xs`, `xs`, `sm`, `md`, `lg`, `xl`, or `2xl` as detailed on the [font size and family](/utilities/font-size-and-family) utility page."
+    description: Change the color of all the list's icons by updating [color] to any [theme](/design-tokens/color/theme-tokens/) or [state](/design-tokens/color/state-tokens/) color token.
+  - variant: "`.usa-icon-list--size-[size]`"
+    description: "Change the size of the text and icon by updating [size] to one of the following theme font sizes: `xs`, `sm`, `md`, `lg`, `xl`, `2xl`, or `3xl` as detailed on the [font size and family](/utilities/font-size-and-family) utility page."
```

Comment:
Just flagging in case we end up needing to update the available theme font sizes list after https://github.com/uswds/uswds/pull/5356

Reply:
Good catch, @mahoneycm!

Reply:
I have added updates to the [documentation PR](https://github.com/uswds/uswds-site/pull/2179) for https://github.com/uswds/uswds/pull/5363. We can go ahead and merge this change in here since this is what is currently accurate, and we can update again once https://github.com/uswds/uswds/pull/5363 is merged.